test/unit/mainnet-controller/Admin.t.sol

```diff
@@ -0,0 +1,48 @@
+// SPDX-License-Identifier: AGPL-3.0-or-later
+pragma solidity ^0.8.21;
+
+import "test/unit/UnitTestBase.t.sol";
+
+contract MainnetControllerFreezeTests is UnitTestBase {
```

Comment:
Should be `MainnetControllerAdminTests`.

Reply:
Fixed